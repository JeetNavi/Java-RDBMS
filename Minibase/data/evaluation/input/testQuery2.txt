Q(x,y,c) :- R(x, y, 'adbs'), R(1, y, c)
