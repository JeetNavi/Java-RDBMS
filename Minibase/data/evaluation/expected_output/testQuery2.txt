8, 9, 'ppls'